src/upload-helper.ts

```diff
@@ -47,12 +47,13 @@ export class UploadHelper {
     bucketName: string,
     filename: string,
     destination?: string,
+    gzip?: boolean,
   ): Promise<UploadResponse> {
     interface UploadOptions {
       gzip: boolean;
       destination?: string;
     }
-    const options: UploadOptions = { gzip: true };
+    const options: UploadOptions = { gzip: gzip === undefined ? true : gzip };
```

Comment:
It feels weird to have this logic here. Can you convert gzip to a boolean in main? Also I believe that an getInput for no input returns a string, so I think the logic should be: 
```
gzip: gzip.toLowerCase() === 'false' ? false : true
```

Reply:
Thanks, updated